### PR TITLE
fix grade filter popover not closing #467

### LINKED ISSUE #467: Can't close grade filter
**Describe the bug**
When you select grade filter you can't close it.

**To Reproduce**

1. Go to https://dreamscholars.org/scholarships
2. Click on the Grade filter.
3. Try closing it. You can't.

**Expected behavior**

It closes when I click away or when I click the Grade filter button again.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by gonzalo1330**
## Motivation and Context

Fixes #467 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] User visible change (users will notice UI or functional changes)

## Previewing Changes

1. Go to https://s4us-pr-485.onrender.com/scholarships
2. Click on the grade level filter
3. Ensure that it closes and opens 
## Checklist:

- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by edwingl10 — DISMISSED** [review]
ahh so enclosing the entire thing in the Button component was preventing it from closing?

File: src/components/GradeLevelFilter.jsx [hunk 1]
```diff
@@ -47,12 +47,15 @@ export default function GradeLevelFilter() {
...
-    <Button
-      variant="outlined"
-      className={classes.button}
-      onClick={openPopover}
-      endIcon={<ArrowDropDownIcon color="primary" />}>
-      Grade
+    <>
+      <Button
+        variant="outlined"
+        className={classes.buttonStyle}
```
Comment (josueetcom) [inline src/components/GradeLevelFilter.jsx:53]:
```suggestion
        className={classes.button}
```

**Review by josueetcom — APPROVED** [review]
The `className` was changed for some reason and messed up the styling. Otherwise LGTM

File: src/components/GradeLevelFilter.jsx (hunk 1 above)
Comment (gonzalo1330) [reply src/components/GradeLevelFilter.jsx:53]: Ok change made

**Comment by josueetcom** [comment]
> ahh so enclosing the entire thing in the Button component was preventing it from closing?

Yep, weird how HTML works sometimes. And of course, no compiler to tell us it's not ok 😓

**Pull request merged**